renovate.json

```
@@ -32,7 +28,16 @@
     },
     {
```

Urgh sorry, I confused json and yaml. Json doesn't take comments in its file content 😢 
Let's make the comment the very first key then.

```suggestion
    {
        "_description": "Ignore pin and digest upgrades. We only want published releases of the SDK to trigger upgrades",   
```

---

That's true i was so confused when you requested😅 i had to improve on the yaml config but i did at least learn something new with the yaml config for references in a json file👍. I guess everything is fine now? Or is there anything else we need to improve of fix?